spec/stat_tracker_spec.rb

```diff
@@ -1,25 +1,35 @@
-require 'spec_helper'
+require 'simplecov'
```

Comment:
Remove lines 1 and 2, only require spec_helper -- as introduced by AJ on zoom.

Reply:
Made the changes!